eng/WpfArcadeSdk/tools/InjectModuleInitializer.targets

```diff
@@ -0,0 +1,423 @@
+        support generating debugging information in the re-assembled binary.
+    -->
+    <When Condition="'$(UseNetCoreILTools)'=='true'">
+
+      <ItemGroup>
+        <PackageReference Include="$(PackageRuntimeIdentifierPrefix).Microsoft.NETCore.ILDAsm" Version="3.0.0-preview6-27710-71" GeneratePathProperty="true" />
+        <PackageReference Include="$(PackageRuntimeIdentifierPrefix).Microsoft.NETCore.ILAsm" Version="3.0.0-preview6-27710-71" GeneratePathProperty="true" />
+        <PackageReference Include="$(PackageRuntimeIdentifierPrefix).Microsoft.NETCore.Runtime.CoreCLR" Version="3.0.0-preview6-27710-71" GeneratePathProperty="true" />
+      </ItemGroup>
+
+      <!-- We use a local RID here since we don't want to change the build layout by setting the actual RID. -->
+      <PropertyGroup>
+        <LocalRuntimeIdentifier Condition="'$(Platform)'!='AnyCPU' and '$(Platform)'!='Win32'">win-$(Platform)</LocalRuntimeIdentifier>
+        <LocalRuntimeIdentifier Condition="'$(Platform)'=='AnyCPU' or '$(Platform)'=='Win32'">win-x86</LocalRuntimeIdentifier>
+      </PropertyGroup>
+
+      <PropertyGroup Condition="'$(LocalRuntimeIdentifier)'=='win-x86'">
+        <ILDAsmPath>$(Pkgruntime_win-x86_Microsoft_NETCore_ILDAsm)\runtimes\$(LocalRuntimeIdentifier)\native\</ILDAsmPath>
+        <ILAsmPath>$(Pkgruntime_win-x86_Microsoft_NETCore_ILAsm)\runtimes\$(LocalRuntimeIdentifier)\native\</ILAsmPath>
+        <CoreCLRPath>$(Pkgruntime_win-x86_Microsoft_NETCore_Runtime_CoreCLR)\runtimes\$(LocalRuntimeIdentifier)\native\</CoreCLRPath>
+      </PropertyGroup>
+
+      <PropertyGroup Condition="'$(LocalRuntimeIdentifier)'=='win-x64'">
+        <ILDAsmPath>$(Pkgruntime_win-x64_Microsoft_NETCore_ILDAsm)\runtimes\$(LocalRuntimeIdentifier)\native\</ILDAsmPath>
+        <ILAsmPath>$(Pkgruntime_win-x64_Microsoft_NETCore_ILAsm)\runtimes\$(LocalRuntimeIdentifier)\native\</ILAsmPath>
+        <CoreCLRPath>$(Pkgruntime_win-x64_Microsoft_NETCore_Runtime_CoreCLR)\runtimes\$(LocalRuntimeIdentifier)\native\</CoreCLRPath>
+      
+      </PropertyGroup>
+
+      <PropertyGroup>
+        <ILToolsPath>$(ArtifactsTmpDir)\iltools\$(PackageRuntimeIdentifierPrefix)\</ILToolsPath>
+        <ILDAsm>$(ILToolsPath)ildasm.exe</ILDAsm>
+        <ILAsm>$(ILToolsPath)ilasm.exe</ILAsm>
+      </PropertyGroup>
+
+    </When>
+  </Choose>
+
+  <PropertyGroup>
+    <TaskFactory>RoslynCodeTaskFactory</TaskFactory>
+    <TaskFactoryAssemblyFile>$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll</TaskFactoryAssemblyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TaskFactoryReference Include="mscorlib" />
+    <TaskFactoryReference Include="netstandard" />
+  </ItemGroup>
+
+  <!--
+      Locate the IL tools included with the highest version of .NET Framework and the .NET Framework SDK installed on the machine.
+  -->
+  <UsingTask TaskName="FindDotNetFrameworkILToolsTask"
```

Comment:
nit: Normally, the second line (`TaskFactory=`) aligns vertically with the previous element (`TaskName=`), like this: 

```xml
<UsingTask TaskName="FindDotNetFrameworkILToolsTask"
           TaskFactory="..."
```